tests/test_suite.py

```diff
@@ -218,21 +218,23 @@ def auto_select_xccdf_id(datastream, bench_number):
 
 def get_datastreams():
     ds_glob = "ssg-*-ds.xml"
-    build_dir_path = [os.path.dirname(__file__), "..", "build"]
-    datastreams = glob(os.path.sep.join(build_dir_path + [ds_glob]))
+    build_dir_path = [os.path.dirname(__file__) or ".", "..", "build"]
+    glob_pattern = os.path.sep.join(build_dir_path + [ds_glob])
+    datastreams = [os.path.normpath(p) for p in glob(glob_pattern)]
     return datastreams
 
 
 def get_unique_datastream():
     datastreams = get_datastreams()
     if len(datastreams) == 1:
         return datastreams[0]
-    msg = ("Unable to auto-detect the datastream, "
-           "expected to find a single one in the build dir, but")
+    msg = ("Autodetection of the datastream file is possible only when there is "
+           "a single one in the build dir, but")
     if not datastreams:
-        raise RuntimeError(msg + " found none.")
+        raise RuntimeError(msg + " there is none.")
     raise RuntimeError(
-        msg + " found multiple ones: {0}".format(datastreams))
+        msg + " there are {0} of them. Use the --datastream option to select "
```

Comment:
The line can get rather long:
```shell
Setting console output to log level INFO
ERROR - Error occurred during options normalization: Autodetection of the datastream file is possible only when there is a single one in the build dir, but there are 28 of them. Use the --datastream option to select e.g. ['../build/ssg-debian8-ds.xml', '../build/ssg-rhel7-ds.xml', '../build/ssg-wrlinux8-ds.xml', '../build/ssg-wrlinux1019-ds.xml', '../build/ssg-ubuntu1404-ds.xml', '../build/ssg-ol8-ds.xml', '../build/ssg-centos6-ds.xml', '../build/ssg-fuse6-ds.xml', '../build/ssg-centos7-ds.xml', '../build/ssg-eap6-ds.xml', '../build/ssg-ubuntu1604-ds.xml', '../build/ssg-ubuntu1804-ds.xml', '../build/ssg-rhv4-ds.xml', '../build/ssg-rhosp13-ds.xml', '../build/ssg-chromium-ds.xml', '../build/ssg-fedora-ds.xml', '../build/ssg-ol7-ds.xml', '../build/ssg-sl6-ds.xml', '../build/ssg-rhel8-ds.xml', '../build/ssg-sle12-ds.xml', '../build/ssg-firefox-ds.xml', '../build/ssg-sl7-ds.xml', '../build/ssg-sle11-ds.xml', '../build/ssg-rhel6-ds.xml', '../build/ssg-opensuse-ds.xml', '../build/ssg-jre-ds.xml', '../build/ssg-ocp3-ds.xml', '../build/ssg-centos8-ds.xml']
```
And in the terminal it actually wraps up.
I like that they short and easy to copy-paste when adding `--datastream` option.